### PR TITLE
add trusted-user-app to examples

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds a new demo app to the examples folder
The app is built with Nextjs and Chakra-UI

A user can connect and check whether they meet some arbitrary trust requirements combining a threshold Passport score and ownership of specific stamps.

The default trust criteria is: 
- Passport score > 20
- Owns a Lens stamp
- Owns a Github stamp

There is a checkbox that can be used to toggle rendering the complete list of the user's stamps

### CODE REVIEW

**Review by 0xZakk — CHANGES_REQUESTED** [review]

File: examples/check-trusted-user-app/app/api/hello/route.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export async function GET(request: Request) {
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply examples/check-trusted-user-app/app/api/hello/route.ts:1]: good point - deleting in next commit

File: examples/check-trusted-user-app/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
+'use client'
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply examples/check-trusted-user-app/app/page.tsx:1]: good point - I forgot to add `page-backup.tsx` to my .gitignore. 
The others are useful though - the tutorial basically takes someone from `boilerplate.tsx` as a starting point to `page.tsx` as a finishing point.

File: examples/check-trusted-user-app/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+This is a simple demo app that shows how to use the Scorer API to retrieve user scores and stamps
+from Gitcoin passport. The app is built in Nextjs using `create-next-app` and Chakra-UI.
+
+## Getting Started
+
+
+Start by creating a new Nextjs project:
+
+```sh
+npx create-next-app passport-app
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply examples/check-trusted-user-app/README.md:13]: yea good point
Another option is to only include page.tsx here and instruct users to create-next-app, copy in page,tsx and then follow the tutorial for settign up their environment variables.

File: examples/check-trusted-user-app/app/page.tsx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+import { useState, useEffect } from 'react'
+import { ethers } from 'ethers'
+import { ChakraProvider, Button, Checkbox, Stack, Badge, SimpleGrid, Heading, Text } from '@chakra-ui/react'
+
+const APIKEY = process.env.NEXT_PUBLIC_GC_API_KEY
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply examples/check-trusted-user-app/app/page.tsx:6]: done (`.env.local-example`)

**Comment by jmcook1186** [comment]
Hi @0xZakk 
removed the unnecessary files, updated the readme, added a .env.local-example.

**Pull request merged**